### PR TITLE
Homework9: Aksana Batyrova

### PR DESCRIPTION
**Pull request by AksanaBatyrova**
HW for lection 9-Functions pt.1

### CODE REVIEW

File: homework_8.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        if x < y < z:
+            x, y = y, z
+        else:
+            difference += 1
+        if difference > 1:
+            return False
+    return x < y
+
+
+print(solution([1, 2, 3]))
```
Comment (Stml89) [inline homework_8.py:56]: better to use assert to check that program works correctly

File: homework_8.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+
+def opposite_number(n, first_number):
+    """
+    This function is counting the opposite number to first_number if they were
+    written around a circle from 0 to n-1
+    n: integer
+    first_number: the number for which needs to find opposite
+
+    return: int
```
Comment (Stml89) [inline homework_8.py:81]: Actually, the program returns float, not int

File: homework_8.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+# Примеры
+# Для n = 10 и first_number = 2 вывод должен быть (n, first_number) = 7.
+
+
+def opposite_number(n, first_number):
+    """
+    This function is counting the opposite number to first_number if they were
+    written around a circle from 0 to n-1
+    n: integer
```
Comment (Stml89) [inline homework_8.py:78]: you have mentioned type, but not the porpoise, as you did in line #79

File: homework_8.py [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+# Примеры
+# validate(4561261212345464) #=> False
+# validate(4561261212345467) #=> True
+
+# Для проверки: https://www.paypalobjects.com/en_GB/vhelp/paypalmanager_help/
+# credit_card_numbers.htm
+
+
+def validate(card_number: int):
```
Comment (Stml89) [inline homework_8.py:103]: what type are you going to return?

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: homework_8.py (hunk 1 above)
Comment (AksanaBatyrova) [reply homework_8.py]: fixed

File: homework_8.py (hunk 3 above)
Comment (AksanaBatyrova) [reply homework_8.py:78]: fixed

File: homework_8.py (hunk 4 above)
Comment (AksanaBatyrova) [reply homework_8.py]: fixed

File: homework_8.py (hunk 2 above)
Comment (AksanaBatyrova) [reply homework_8.py]: fixed

**Pull request merged**